.github/workflows/build.yml

```diff
@@ -54,10 +54,10 @@ jobs:
 
       - name: Generate code from Protobuf definitions
         run: |
-          yarn
           mkdir ./src/proto
-          yarn pbjs -t static-module -w es6 -o ./src/proto/config.js ./proto/config.proto
-          yarn pbts -o ./src/proto/config.d.ts ./src/proto/config.js
+          cd src
+          npx buf generate
+          cd ..
```

Comment:
If you change output directory to `src` in `buf.gen.yaml` and move it to the root of the repo, you can then replace these lines with `npx buf generate --path proto`, and you can also get rid of the `mkdir` command on L57.